gitops/snapshot_predicate_test.go

```diff
@@ -137,4 +188,16 @@ var _ = Describe("Predicates", Ordered, func() {
 			Expect(instance.Delete(contextEvent)).To(BeFalse())
 		})
 	})
+
+	Context("when testing IntegrationSnapshotChangePredicate predicate", func() {
```

Comment:
Maybe a couple of more tests could go here, such as:

- Check if the predicate returns true when the old Snapshot doesn't have annotation and the new one does
- Check if the predicate returns false when both Snapshots have the same annotation content